learn-more.html

```diff
@@ -1,138 +0,0 @@
-            <a href="blog-post.html" class="buttonS">July 30</a>
-            <div class="post-head text-medium">Just Another Post</div>
-            <div class="post-content text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam dictum non mauris et lacinia. Quisque facilisis nibh magna, condimentum efficitur neque maximus in. Aenean ultrices gravida ante, eget tincidunt justo sollicitudin in.
-            </div>
-          </div>
-          <div class="post">
-            <a href="blog-post.html" class="buttonS">July 28</a>
-            <div class="post-head text-medium">And Another Post</div>
-            <div class="post-content text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam dictum non mauris et lacinia. Quisque facilisis nibh magna, condimentum efficitur neque maximus in. Aenean ultrices gravida ante, eget tincidunt justo sollicitudin in. 
-            </div>
-          </div>
-          <div class="post">
-            <a href="blog-post.html" class="buttonS">July 26</a>
-            <div class="post-head text-medium">One More Post</div>
-            <div class="post-content text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam dictum non mauris et lacinia. Quisque facilisis nibh magna, condimentum efficitur neque maximus in. Aenean ultrices gravida ante, eget tincidunt justo sollicitudin in.
-            </div>
-          </div>
-        </div>
-
-        <div id="of-note">
-          <div class="note-chunk">
-            <h1 class="note-head text-medium">Something of Note</h1>
-            <div class="note-content">
-              <div id="note-box"></div>
-              <div class="note-text text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas cursus consectetur enim. Phasellus pellentesque sit amet risus vehicula rhoncus. Suspendisse eget ullamcorper tortor. Aliquam id quam porta, aliquam urna sed, ultrices augue. Integer ut pharetra ligula, a gravida orci. Duis dui massa, tincidunt pulvinar diam ac, gravida suscipit turpis. Phasellus.</div>
-            </div>
-            <a href="learn-more.html" class="buttonL">LEARN MORE</a>
-          </div>
-          <div class="note-chunk">
-            <h1 class="note-head text-medium">Something of Note</h1>
-            <div class="note-content">
-              <div id="note-box"></div>
-              <div class="note-text text-small">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas cursus consectetur enim. Phasellus pellentesque sit amet risus vehicula rhoncus. Suspendisse eget ullamcorper tortor. Aliquam id quam porta, aliquam urna sed, ultrices augue. Integer ut pharetra ligula, a gravida orci. Duis dui massa, tincidunt pulvinar diam ac, gravida suscipit turpis. Phasellus.</div>
-            </div>
-            <a href="learn-more.html" class="buttonL">LEARN MORE</a>
-          </div>
-        </div>
-
-      </div>
-
-    </section>
-    
-<!--- - - - - - footer - - - - -->

```

Comment:
good comment for sectioning out areas of your code. A few more comments throughout would help.